BrickSelect/README.md

```diff
@@ -0,0 +1,17 @@
+# BrickSelect
+
+BrickSelect is a web component from PROGRAP developers who wanted an reusable and extendable drop-in replacement selector element.
```

Comment:
BrickSelect is a ~~web component from~~  **custom element by** PROGRAP ~~developers who wanted~~ **providing a** reusable and extendable drop-in replacement **select input** element.